feature-proxy-manager/src/main/java/com/kinandcarta/create/proxytoggle/manager/settings/DeviceSettingsManager.kt

```diff
@@ -0,0 +1,46 @@
+package com.kinandcarta.create.proxytoggle.manager.settings
```

Comment:
do you have layer modules or packages? Since this class is aware of Android context it should be in the view layer

Reply:
Uhm, good point. I don't think `view` is the correct layer. I agree it could have a better naming, but this is not a view. What about having a `framework` or `device` layer? I agree should be in the outer ring (in terms of Clean Arch), same level as Views but not in the same package necessarily

Reply:
It is tied to the view layer as it is aware of Android context. Something doesn't have to be a view component to be in the view layer.  You can create a separate package or module for this type of class that doesn't reference a view class but is aware of the Android framework, but it should be made clear that it is part of the view layer. If the naming view layer seems confusing (it is a convention used to describe all logic related to Android classes/framework), then rename it to androidframework or something. The convention in clean architecture is view (Android related, iOS related, Flutter related, Web related) -> presentation (MVVM, MVI etc, Models, Mappers etc) -> domain (UseCase, Models, Repository Interface) -> data (Repository Implementation, Data Sources etc)

Reply:
You got it, sir! Not convinced of naming `view` to that package but it's a fair point

Reply:
I have seen some call it ui. You could name it whatever. The important thing is to have a clear separation for classes that should be in the same layer